CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift

```diff
@@ -65,6 +65,32 @@ class ContentResponseHandlerSpec: QuickSpec
                     }
                 }
             }
+            
+            it("Should return Connected contents after login")
+            {
```

Comment:
It depends on the account. I checked it with another one (PM me for details), and everyghing was ok